src/Bucket/Bucket.jl

```diff
@@ -238,6 +259,11 @@ function BucketModel(;
     atmosphere::ATM,
     radiation::RAD,
 ) where {FT, PSE, ATM, RAD}
+    if typeof(parameters.albedo) == BulkAlbedoMap{FT}
```

Comment:
This seems like it could maybe be done using multiple dispatch instead, though I'm not sure how we could access `parameters.albedo` in the function signature itself.

Reply:
We can use UnionSplitting
```suggestion
    if parameters.albedo isa BulkAlbedoMap
```
which is equivalent to multiple dispatch for small (i.e., a few) unions, ref: https://julialang.org/blog/2018/08/union-splitting/

Reply:
nice!